src/db/queries/transactionGroup.ts

```diff
@@ -360,10 +360,107 @@ export default function useTransactionGroup() {
     }
   }
 
+
+  const getTransactionsByGroupId = async ({ id }: { id: number}) => {
+    setLoading(true)
+    setError(null)
+
+    try
+    {
+      // Abfrage aus Datenbank
+      const rows = await db
+          .select({
+            group: {
+              id: transactionGroupTable.id,
+              name: transactionGroupTable.name,
+              note: transactionGroupTable.note,
+              date: transactionGroupTable.date,
+            },
+            tx: {
+              id: transactionTable.id,
+              name: transactionTable.name,
+              amount: transactionTable.amount,
+            },
+            term: {
+              id: categoryTermTable.id,
+              term: categoryTermTable.term,
+            },
+            category: {
+              id: categoryTable.id,
+              name: categoryTable.name,
+              emoji: categoryTable.emoji,
+              color: categoryTable.color,
+            },
```

Comment:
```suggestion
            transaction: {
              id: transactionTable.id,
              name: transactionTable.name,
              amount: transactionTable.amount,
              categoryTermId: categoryTermTable.id,
              categoryTerm: categoryTermTable.term,
              categoryId: categoryTable.id,
              categoryName: categoryTable.name,
              categoryEmoji: categoryTable.emoji,
              categoryColor: categoryTable.color,
            },
```
Muesstest die Transaktions und Kategorie-Daten auf ein Objekt laden koennen, dann sparst du dir das spaetere mapping.